src/Modules/OrchardCore.Commerce/Assets/Styles/credit-card-form.scss

```diff
@@ -14,11 +14,6 @@
 .pay-button {
     width: 50%;
     color: white;
```

Comment:
Do you still need this?

Reply:
Good point! The `color` is not needed anymore, but `width` is, because the default button is just too small.

Reply:
Yes, I just meant the `color`.